packages/events/src/event.rs

```diff
@@ -71,3 +73,97 @@ fn decode_event_attribute(attribute: &EventAttribute) -> Result<(String, String)
 fn base64_to_utf8(base64_str: &str) -> std::result::Result<String, DecodingError> {
     Ok(STANDARD.decode(base64_str)?.then(String::from_utf8)?)
 }
+
+pub fn get_contract_address(event: &Event) -> Option<AccountId> {
+    let contract_address = match event {
+        Event::Abci {
+            event_type: _,
+            attributes,
+        } => {
+            if let Some(address) = attributes.get("_contract_address") {
+                return serde_json::from_value::<AccountId>(address.clone()).ok();
+            }
+            None
+        }
+        _ => None,
+    };
+    contract_address
+}
```

Comment:
```suggestion
pub fn contract_address(event: &Event) -> Option<AccountId> {
    match event {
        Event::Abci {
            event_type: _,
            attributes,
        } => attributes
            .get("_contract_address")
            .and_then(|address| serde_json::from_value::<AccountId>(address.clone()).ok()),
        _ => None,
    }
}
}
```
rust convention is to not use `get_` for getters and instead just call out the thing you're getting